src/ProblemSolving/BreakingTheRecords.java

```diff
@@ -0,0 +1,29 @@
+package ProblemSolving;
+
+public class BreakingTheRecords {
+    public static int[] breakingRecords(int[] score){
+        int minimumCount = 0;
+        int maximumCount = 0;
+        for (int i = 0; i < score.length; i ++){
+            int minimum = score[0];
+            int maximum = score[0];
+            if (minimum == 0 & maximum == 0){
+                minimum = score[i];
+                maximum = score[i];
+            }
+            if(minimum > score[i]){
+                minimum = score[i];
+                minimumCount++;
+            }
+            if(maximum < score[i]){
+                maximum = score[i];
+                maximumCount++;
```

Comment:
If the above block can be omitted minimumCount++; can be put outside of the conditions.